src/com/ua/RD/Homework5/Main.java

```diff
@@ -0,0 +1,26 @@
+package com.ua.RD.Homework5;
+
+public class Main {
+    public static void main(String[] args) {
+        int a = 10;
+        int b = 12;
+
+        boolean aGreaterPair = a > b && a != b && a % 2 == 0;
+        boolean bGreaterPair = a < b && a != b&& b % 2 == 0;
+        boolean aGreaterNonPair = a > b && a != b && a % 2 != 0;
+        boolean bGreaterNonPair = a < b && a != b&& b % 2 != 0;
+
+
+        if (aGreaterPair) {
+            System.out.println("A is pair and greater than B");
+        } else if (bGreaterPair) {
+            System.out.println("B is pair greater than A");
```

Comment:
B is pair "AND" greater than A

Reply:
it works. but you combined 2 tasks in 1. I will accept it, but please separate it to 2 different tasks

Reply:
> it works. but you combined 2 tasks in 1. I will accept it, but please separate it to 2 different tasks

updated and masseged in LMS